### PR TITLE
Avatar placeholders

### LINKED ISSUE #1562: Avatar placeholders
Right now we had an issue when avatars are unavailable for some reason and we didn't display the placeholder that does not depend on a storage. We need to display the placeholder image if real avatar in not fetched or fetched with an exception

### PR DESCRIPTION
**Pull request by Cheshiriks**
Closes #1562

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt [hunk 1]
```diff
@@ -203,6 +208,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                 isEditDisabled = true
                 selfRole = highestRole
                 usersInOrganization = users
+                avatar = organizationLoaded.avatar?.let { "/api/$v1/avatar$it" } ?: run { "img/undraw_profile.svg" }
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt:211]: while the latest string is in `run`?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt [hunk 2]
```diff
@@ -523,14 +529,14 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
...
-                        }
-                            ?: run {
-                                "img/company.svg"
-                            }
+                        src = state.avatar
                         height = 100.0
                         width = 100.0
+                        onError = {
+                            setState {
+                                avatar = "img/undraw_image_not_found.png"
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt:537]: maybe extract this variable to a separate constant?

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt]: Fixed

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt]: Added

**Pull request merged**